src/Tizen.NUI/src/public/BaseComponents/Style/Selector.cs

```diff
@@ -158,42 +158,49 @@ public T DisabledSelected
         [EditorBrowsable(EditorBrowsableState.Never)]
         public T Other
         {
-            get;
-            set;
+            get => (T)Find(x => x.State == ControlState.Other).Value;
+            set => Add(ControlState.Other, value);
         }
         /// <summary>
         /// Get value by State.
         /// </summary>
         /// <since_tizen> 6 </since_tizen>
         /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public T GetValue(ControlStates state)
+        public T GetValue(ControlState state)
         {
             if(All != null)
```

Comment:
In the future, we may think of optimizing speed by adding private field `StateValuePair all;`

Because `All` is the most popular state.
- Selector calls `All` frequently in `GetValue` (which is called everytime when the View's control state has changed).
- The `All` may used a lot because when a user does not need a selector with multiple state value, `All` is used.
Note that the below examples are the same.
```
TextLabelStyle a = new TextLabelStyle
{
    TextColor = "Yellow",
}
```
```
TextLabelStyle a = new TextLabelStyle
{
    TextColor = new Selector<Color>{ All = "Yellow" };
}
```